llvm/test/Transforms/LoopVectorize/AArch64/sleef-calls.ll

```diff
@@ -1,4 +1,4 @@
-; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --filter "(_)|(cos|expf?\(|exp2|exp10|fmod|gamma|log|pow|sin|sqrt|tan)|(ret)" --version 2
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --filter "call.*(cos|^exp|mod|gamma|log|pow|sin|sqrt|tan)" --version 4
 ; RUN: opt -vector-library=sleefgnuabi -passes=inject-tli-mappings,loop-vectorize,simplifycfg -prefer-predicate-over-epilogue=predicate-dont-vectorize -force-vector-interleave=1 -S < %s | FileCheck %s --check-prefix=NEON
```

Comment:
could you remove  the -prefer-predicate-over-epilogue=predicate-dont-vectorize from here?

Reply:
It looks like the ArmPL tests are having more coverage maybe it is wort to copy them here and just modify the run lines?

The sleef tests are much older, added in 2018, by https://reviews.llvm.org/D53927
I added the ArmPL tests myself in https://reviews.llvm.org/D154508, but there is no need to have them written differently.
 The difference is only in the mappings used, I think that unifying them may have sense (but still keep them in separate file to avoid long files), and since you are doing a bit of refactoring maybe you could do it?

Reply:
Ok I tried this and didn't seem a good idea. The SLEEF mappings are fewer than the ArmPL ones.

Reply:
I think having the extra tests does not hurt, that means most likely that more SLEEF mappings can be added, as at the moment the ArmPL mappings in LLVM contain only the basics.

The reason why I prefer the ArmPL tests is that they are easier to read, the use pointers to src and dst of the function calls, and do not any extra not needed complexity to convert data to required format to pass it to the function call.

Reply:
done